setup.cfg

```diff
@@ -137,7 +137,7 @@ install_requires =
     tabulate>=0.7.5, <0.9
     tenacity>=6.2.0
     termcolor>=1.1.0
-    typing-extensions>=3.7.4;python_version<"3.8"
```

Comment:
I think we should add a comment here that we can remove this dependency when one of two things happen:

1) We drop Pyhon 3.7
2) We add min airflow version for all providers to be 2.2 - in which case we can replace it with compat.

Reply:
BTW. The 2. might happen in a few months, so it's good to note it to remember and remove it eventually.

Reply:
Addressed in latest commit